src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -70,6 +98,7 @@ def fetch_alert_rule(request: Request, organization, alert_rule):
                 # Delete meta fields
                 del action["_sentry_app_installation"]
                 del action["_sentry_app_component"]
+                del action["_sentry_app"]
```

Comment:
line 57 ensures "_sentry_app_installation" and ""_sentry_app_component" exist, but don't see code that would guarantee that `del action["_sentry_app"]` won't throw a KeyError

Reply:
I ended up having to remove the `_sentry_app` key entirely from `get_attrs` as the `AlertRuleSerializer` is used in other notification delivery contexts as well. I thought having an additional loop in the endpoint logic was less of a cludge than having the kludge in all the places the serializer is used.